src/modules/accommodations/controllers/accommodations.controller.ts

```diff
@@ -11,8 +13,23 @@ export class AccommodationsController {
   }
 
   @Get()
-  findAll() {
-    return this.AccommodationService.findAll();
+  @ApiOperation({
+    summary: 'Get list of accommodations that match filtering and sorting',
+  })
+  @ApiResponse({
+    status: 201,
```

Comment:
Success status for GET request should not be 201, it should be 200

Reply:
fixed